coremltools/converters/mil/frontend/torch/test/test_torch_ops.py

```diff
@@ -2798,7 +2798,21 @@ class TestEinsum(TorchBaseTest):
              "abc,cde->abde",
              "btnh,bfnh->bnft",
              "bnft,btnh->bfnh",
-             "abcd,cde->abe"],
+             "abcd,cde->abe",
```

Comment:
This list is getting out of hand.

Please group these entries by length. Then alphabetize each group.

Reply:
Should I sort added testcases only or all cases including existing test cases?

I think existing test cases order make sense since it is matched with non-generic eisum implementation.
https://github.com/apple/coremltools/blob/35693696123b538ef9a5de326ee2322f5aac1c68/coremltools/converters/mil/frontend/_utils.py#L59